blog/content/post/2023-09-12-simplifying-firmware-update.md

```diff
@@ -0,0 +1,112 @@
+---
+title: 'Simplifying Firmware Updates in Dasharo'
+abstract: "Updating your firmware is an important part of keeping your device
+          secure and reliable. Making this process as easy and reliable as
+          possible is, therefore, a big focus for Dasharo. In this article we'll
+          dive into the latest feature that make that process a bit easier, and
+          talk about where we want to go from here."
+cover: /covers/dasharo-sygnet.svg
+author: michal.kopec
+layout: post
+published: true
+date: 2023-09-12
+archives: "2023"
+
+tags:
+  - coreboot
+  - UEFI
+categories:
+  - Firmware
+
+---
+
+# Introduction
+
+The word *firmware* comes from the fact that it sits somewhere in between
+*soft*ware and *hard*ware. This term refers to the fact that firmware provides
+an abstraction layer for hardware, so that software (e.g. OS) can make use of it
+in a more generic manner.
+
+Firmware also sits between hardware and software in a different category, which
+is how hard it is to modify. While software is easy to modify and update,
+hardware is set in stone and cannot be fixed so easily (at least without
+recalling devices and replacing components). Firmware, again, sits somewhere in
```

Comment:
What does recalling devices mean here?

Reply:
As in the manufacturer takes them back for repair / replacement, see https://support.lenovo.com/us/en/solutions/ht004883-battery-recall or https://en.wikipedia.org/wiki/Volkswagen_emissions_scandal#Vehicle_recall_and_consequences